gazelle/python/testdata/directive_python_default_visibility/test6_default_label/subpkg/BUILD.out

```diff
@@ -0,0 +1,10 @@
+load("@rules_python//python:defs.bzl", "py_library")
+
+# Reset the default visibility to the default for all child packages.
+# gazelle:python_default_visibility DEFAULT
```

Comment:
What if we set `# gazelle:python_visibility` together with `# gazelle:python_default_visibility`?

Reply:
How about a test case that uses `# gazelle:python_visibility` on a sub-package?